config/v1/types_feature.go

```diff
@@ -125,18 +125,65 @@ type FeatureGateDetails struct {
 	// disabled is a list of all feature gates that are disabled in the cluster for the named version.
 	// +optional
 	Disabled []FeatureGateAttributes `json:"disabled"`
+	// renderedMinimumComponentVersions are the component versions that the feature gate list of this status were rendered from.
+	// Currently, the only supported component is Kubelet, and setting a required minimum kubelet component will set the
+	// minimumKubeletVersion field in the nodes.config.openshift.io CRD.
```

Comment:
Still need to fix "will set the", this implies that something being set here implicitly sets the nodes config and I don't think we want to imply that

Reply:
oops done